1.10/Dockerfile

```diff
@@ -8,7 +8,12 @@ FROM centos/s2i-core-centos7
 EXPOSE 8080
 EXPOSE 8443
 
-ENV NGINX_VERSION=1.10
+ENV NAME=nginx \
+    NGINX_VERSION=1.10 \
+    NGINX_SHORT_VER=110 \
+    VERSION=0 \
+    RELEASE=1 \
+    ARCH=x86_64
```

Comment:
Hard-coding arch is not a good idea, it would make us trouble once there are multi-arch containers.

Reply:
Removed.

Reply:
@phracek So no need to have ARCH env var.

Also we are removing `release` labels from CentOS/RHEL images.

Reply:
Why? https://fedoraproject.org/wiki/Container:Guidelines#LABELS contains release label as required. Although, it will be still 1.

Reply:
The way I see it the Fedora version of release label also has a DISTTAG suffix added (that expands to something like f26container) which is why the release label has to be present. We do not have anything like that in RHEL/CentOS so we can just remove it completely and let the build system increment this.

Also the guidelines say that the release shout NOT stay at 1:
"release: Populated from the ENV variable. Should start at 1, and increment every time the image changes."

Reply:
Increasing RELEASE should be done by each change. Yes, Your team should increase it. I will do it over all my changes in nginx container.

Reply:
Increasing the release in upstream sources is really not necessary, because here on github we generally don't know how many times we needed to increase it downstream before and there is no reason to keep it in sync. If the release is necessary for Fedora Dockerfile, fine, let's keep it there for the sake of having the line on a correct place, but in Dockerfiles for other distros, it's really not necessary now, because internal OSBS bumps it automatically and we don't use it at all for CentOS images.